x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -695,3 +695,115 @@ emp_no:integer | languages:integer | gender:keyword | max_lang:integer | y:keywo
          10012 |                 5 | null           | 5                | null
          10014 |                 5 | null           | 5                | null
 ;
+
```

Comment:
Most of the following tests (except one) fail with the same error. This is likely due to something that @alex-spies caught in a [previous review](https://github.com/elastic/elasticsearch/pull/122257#discussion_r1951221550), meaning too much is sent to the data nodes for processing, instead of being done on the coordinator. This is also tied to scenarios where multiple `inlinestats` commands are used in a query.